src/api/v2/orders.rs

```diff
@@ -122,6 +149,7 @@ mod tests {
   }
 
   #[test(tokio::test)]
+  #[serial]
```

Comment:
Should we have a separate commit for this change? Also, can we use a key such as `order_aapl` to not serialize it with respect to unrelated functionality?

For my education, can you explain what the conflict is you saw, please?

Reply:
The conflict I was seeing had to with the `list_orders` test running in parallel. Since my test and that one were purchasing `aapl`, there were more open orders than what my test was expecting. I have updated my tests to use a different stock, which should prevent conflicts.

Reply:
Sounds good. That will work as well, we just have to be careful about not using too many different and too insignificant stocks -- otherwise that may just introduce more maintenance burden down the line when they get purchased, go out of business, or vanish for another reason.

Reply:
Noted - I used big name stocks other than aapl.